10.1 updates (#92)
TexasInstruments/processor-sdk-doc: source/system/Demo_User_Guides/TI_Apps_Launcher_User_Guide.rst

   .. Image:: /images/ti-apps-launcher-j7.png
      :height: 400

> |

Reviewer: ```suggestion

```
(WARNING/2) Line block ends without a blank line.

3. Once you enter the necessary values for both HTTPS and No proxy, click on `Set Proxy` to set the proxy configuration.

> Using terminal

Reviewer 2: Sections use ----, not ____. https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

Author: Noted will update

______________

.. ifconfig:: CONFIG_sdk in ('SITARA')

    The terminal button on the left panel can be used to open a terminal application, which is
    based on QMLtermwidget.

>     .. Image:: /images/terminal.png

Reviewer 3: For all Image references,

Checkout https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace for indentations of height attribute with Image directive

      :height: 400

> .. ifconfig:: CONFIG_sdk not in ('SITARA')

Reviewer 2: Why not put this whole section inside the ifconfig rather than suggest that it is not available?

Author: @AashvijShenai Not sure but i am observed that sphinx does not support declaring a new subsection inside a ifconfig. I added a test section
```
.. ifconfig:: CONFIG_part_variant in ('AM62X', 'AM62PX')                                                                                                  

    Test Menu                                                                    
     ^^^^^^^^^                                                                                                                                                     
    This is a test section
```
The test Menu appeared as a new section after build. (And not a sub-section which is required). If you can try it on your end as well.

>    This section is not valid for this platform.

Reviewer 3: ```suggestion
   This section is not applicable for this platform.
```

In entire page

Using Wi-Fi Demo

> ________________

Reviewer 2: Section marker

.. ifconfig:: CONFIG_part_variant in ('AM62X', 'AM62PX')

    The *Wifi* button on the left panel allows you to connect to an external Wi-Fi network.

>     Ensure the required overlay is applied and the WiFi card is connected to SK-EVM. For more

Reviewer 3: ```suggestion
    Ensure the required overlay is applied and the M.2 WiFi card is connected to SK-EVM. For more
```

    information, refer to :ref:`enable_m2cc3301`.

    Click the toggle button to turn on Wi-Fi.

    .. Image:: /images/wifi_home.png
        :height: 400

    * Click on the *Refresh* button to scan for available SSIDs.
    * Select an SSID from the list, then enter the passphrase and Wi-Fi security type.
    * For certain security types, you may need to provide additional details.
    * After entering all the required info, click on *Connect*.

    .. Image:: /images/wifi_ssid.png
        :height: 400

    If the all entered details are correct, device will be connected to the selected Wi-Fi
    network. If needed, click *Disconnect* to disconnect from the network and return to
    SSID selection menu.

    .. Image:: /images/wifi_connected.png
        :height: 400

>     .. ifconfig:: CONFIG_part_variant in ('AM62X')

Reviewer 3: Indentation for all ifcongig, note, Image directive doesn’t follow the CONTRIBUTING guidelines- https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace

      .. note::

         This section is not applicable for AM62xLP and AM62xSIP.

> .. ifconfig:: CONFIG_part_variant not in ('AM62X', 'AM62PX')

Reviewer 2: Same thing as above, Why not define the whole section inside .. ifconfig:: CONFIG_part_variant in ('AM62X', 'AM62PX')

Author: Same comment as above.

   This section is not valid for this platform.

Power Menu
__________

1. The top right corner of the TI Apps Launcher is dedicated for the Power Menu.

.. ifconfig:: CONFIG_part_variant in ('AM62X', 'AM62PX')

>     2. The Power Menu supports Shutdown, Reboot, Suspend to RAM (For NON RT Build) and Exit (from TI Apps Launcher) functions.

Reviewer 3: ```suggestion
    2. The Power Menu supports Shutdown, Reboot, Suspend to RAM (For Linux only) and Exit (from TI Apps Launcher) functions.
```
